packages/bierzo-wallet/src/components/Wallet/ContinueButton/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { Button, withStyles } from '@material-ui/core';
+import React from 'react';
+
+const StyledButton = withStyles({
+  root: {
+    '&&': {
```

Comment:
What it does the double ampersand?

Reply:
A single ampersand is replaced by the compiled parent selector. The double ampersand is used to increase the specificity of the rules by having the compiled parent selector two times.

I found that without it I couldn't override the default rules of Material UI. For instance, I couldn't make the avatar's default color yellow.

Reply:
Removed the double ampersand in [c294569](https://github.com/iov-one/ponferrada/pull/79/commits/c294569bcd0b16f865ce6527ed5af11185a3bcd5).